scripts/meetingbaas.py

```
@@ -709,13 +1154,25 @@ def cli() -> None:
     # Parse the full persona payload from the parent process; main() uses it
     # directly when it carries a prompt (dynamic personas are never on disk).
     persona_data = None
-    if args.persona_data_json:
+    if args.persona_data_file:
+        try:
+            with open(args.persona_data_file, "r") as f:
```

Ruff (0.15.20)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)